client/src/App.js

```diff
@@ -45,7 +45,15 @@ function App() {
         <LinearGradient width="100px" />
         <LinearGradient width="300px" />
         <LinearGradient width="500px" />
-        <nav className="nav-bar">
+        <nav className="fixed w-full z-10 flex items-center justify-between h-16 backdrop-filter backdrop-blur-lg border-b border-gray-200">
```

Comment:
Let's drop the `border-b` and `border-gray-200` for now and add in `bg-gray-600 bg-opacity-50` to give it a little darker grey color